InvenTree/order/views.py

```diff
@@ -572,7 +576,58 @@ def post(self, request, *args, **kwargs):
 class PurchaseOrderUpload(FileManagementFormView):
     ''' PurchaseOrder: Upload file, match to fields and parts (using multi-Step form) '''
 
+    # overriden classes
+    class MyMatch(cm_forms.MatchItem):
```

Comment:
I don't know if that's good Django practice, nesting a form class definition inside a form view..? What do you think @SchrodingersGat?

Reply:
Maybe better declaring it in `order/forms.py`?

Reply:
Also the name should be `OrderMatchItem`, it should be explicit

Reply:
@eeintech I would not see this as a normal form - it just overrides one function. I could refactor it to look more like a override but that makes it more difficult to read.